Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionTester.swift

```diff
@@ -192,7 +182,7 @@ final class NetworkProtectionConnectionTester {
 
     // MARK: - Testing the connection
 
-    func testConnection() async throws {
+    func testConnection() async {
```

Comment:
This function wasn't throwing anything, but it was causing a chain of `try` statements above it, so removing `throws` here lets us clean up a bit of code.

Reply:
Really nice, thanks for taking the time to clean up.